docs/what-is-ethereum/index.md

```diff
@@ -37,28 +44,16 @@ These decentralized applications (or “dapps”) gain the benefits of cryptocur
 - **Games** where you own in-game assets, and can even make real money
 - **And much,** much more.
 
-**The Ethereum community is the largest and most active blockchain community in the world.** It includes core protocol developers, cryptoeconomic researchers, cypherpunks, mining organizations, ETH holders, app developers, ordinary users, anarchists, fortune 500 companies, and, as of now, **you**.
+### The Ethereum community is the largest and most active blockchain community in the world. {.mt-3}
 
-**There is no company or centralized organization that controls Ethereum.** Ethereum is maintained and improved over time by a diverse global community of contributors who work on everything from the core protocol to consumer applications. This website, just like the rest of Ethereum, was built - and continues to be built - by a collection of people working together.
+It includes core protocol developers, cryptoeconomic researchers, cypherpunks, mining organizations, ETH holders, app developers, ordinary users, anarchists, fortune 500 companies, and, as of now, **you**.
 
-**Welcome to Ethereum.**
+### There is no company or centralized organization that controls Ethereum. {.mt-3}
 
-**Not sure where to go next?**
+Ethereum is maintained and improved over time by a diverse global community of contributors who work on everything from the core protocol to consumer applications. This website, just like the rest of Ethereum, was built - and continues to be built - by a collection of people working together.
 
-- Want to get started using Ethereum? [ethereum.org/dapps](/dapps/)
-- Curious to learn more about Ethereum and its technology? [ethereum.org/learn](/learn/)
-- Are you a developer interested in building on ethereum? [ethereum.org/developers](/developers/)
+## Where to go next:
 
-**Looking for more beginner resources about Ethereum?**
+<ListCard :items="$page.frontmatter.lastresources" />
```

Comment:
![Image 2020-03-30 at 9 57 29 AM](https://user-images.githubusercontent.com/8097623/77940640-da700e80-726d-11ea-99f8-3ed5b4ef39da.png)

In my opinion, this isn't the best use of this list format. To me this makes the most sense when items are sequential. Having the items be labeled 1, 2 & 3 gives the sense that I should visit these resources in order but in this situation, I don't think that's necessary or even wanted (e.g. many people may never want to check out how to "Build on Ethereum as a Developer").

I think it'd be more fitting to have full cards where users choose their own adventure of sorts, similar to the /build page:
![Image 2020-03-30 at 9 57 51 AM](https://user-images.githubusercontent.com/8097623/77940709-ee1b7500-726d-11ea-825f-abc00293ff0a.png)
Thoughts?